File: RB101_Programming_Foundations/small_problems/Easy2/Run2/tip_calculator.rb
print "What is the bill? "
bill = gets.chomp.to_f
print "What is the tip percentage? "
tip = gets.chomp.to_f / 100

tip = bill * tip
total = bill + tip

puts "The tip is $#{format("%0.2f", tip)}" 
puts "The total is $#{format("%0.2f", total)}"

File: js210_fundamentals_of_JS_Programming/problem_sets/easy1/tip_calculator.js
let readlineSync = require('readline-sync');
let bill = readlineSync.question('What is the bill? ', { limit: /[\d.]/ });
bill = parseFloat(bill);
let tipPercent = readlineSync.question('What is the tip tip percentage? ', { limit: /[\d.]/ });
tipPercent = parseFloat(tipPercent) / 100;
let tip = tipPercent * bill;
let total = bill + tip;

console.log(`The tip is ${tip.toFixed(2)}`);
console.log(`The total is ${total.toFixed(2)}`);
